### ISSUE TITLE
om fails to authenticate if password has double quotes in it.

### ISSUE THREAD

**Issue by dforbes-pivotal** [pivotal-cf/om#573]
Using om version 5.0.19 it has been noticed that using double quotes is resulting in a failed authentication.  Escaping the character is resulting in success.  

For example:=
```
root@4f12ff08-3b58-4e13-4426-f2cff59398de:/tmp/build/35bf6b7a# env  | grep -i mu3
OM_VAR_om_decryption_pwd=+qh4CCbaMu3"
root@4f12ff08-3b58-4e13-4426-f2cff59398de:/tmp/build/35bf6b7a# om --env env/vars/dev02/env.yml configure-director --config config/configs/director/director-2.10.24.yml --vars-file config/vars/global/vars.yml --vars-file config/vars/global/secrets.yml --vars-file config/vars/env_global/vars.yml --vars-file config/vars/env_global/secrets.yml --vars-file config/vars/dev02/vars.yml --vars-file config/vars/dev02/secrets.yml --vars-file vars/input_versions/dev.yml
2021/12/22 14:07:34 could not list installations: could not make api request to installations endpoint: could not send api request to GET /api/v0/installations: could not unlock ops manager, check if the decryption passphrase is correct

root@4f12ff08-3b58-4e13-4426-f2cff59398de:/tmp/build/35bf6b7a# export OM_VAR_om_decryption_pwd=+qh4CCboMu3\\\"
root@4f12ff08-3b58-4e13-4426-f2cff59398de:/tmp/build/35bf6b7a# env  | grep -i mu3
OM_VAR_om_decryption_pwd=+qh4CCbaMu3\"
root@4f12ff08-3b58-4e13-4426-f2cff59398de:/tmp/build/35bf6b7a# om --env env/vars/dev02/env.yml configure-director --config config/configs/director/director-2.10.24.yml --vars-file config/vars/global/vars.yml --vars-file config/vars/global/secrets.yml --vars-file config/vars/env_global/vars.yml --vars-file config/vars/env_global/secrets.yml --vars-file config/vars/dev02/vars.yml --vars-file config/vars/dev02/secrets.yml --vars-file vars/input_versions/dev.yml
started configuring director options for bosh tile
finished configuring director options for bosh tile
started configuring availability zone options for bosh tile
successfully fetched AZs, continuing
finished configuring availability zone options for bosh tile
started configuring network options for bosh tile
finished configuring network options for bosh tile
started configuring network assignment options for bosh tile
unable to set network assignment for director as it has already been deployed
finished configuring network assignment options for bosh tile
started configuring resource options for bosh tile
finished configuring resource options for bosh tile
root@4f12ff08-3b58-4e13-4426-f2cff59398de:/tmp/build/35bf6b7a#
```

**Comment by anEXPer** [comment]
How is that env var getting set in that environment, in the first example?

**Comment by dforbes-pivotal** [comment]
Hi, in the example the environment variable is simply set via an export of the variable name.  This is a slightly simplified example driven from the command line but the issue initiallly showed up via a failed authentication using platform automation from concourse.  Investigation via the hijacked container demonstrated the password with double quotes issue.

**Comment by jaristiz** [comment]
Hi @dforbes-pivotal 

We could reproduce the problem and have it narrowed to the way we handle the decryption passphrase variable.
We are working on a fix right now and keep you posted on the progress.

**Comment by jaristiz** [comment]
Fixed on version `7.4.2`
